### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by lagartoverde**

### CODE REVIEW

**Comment by lagartoverde** [comment]
This PR solves
https://binnenland.atlassian.net/browse/GN-1972?atlOrigin=eyJpIjoiMmE1ZTY2NzNiZjA3NDM3YmE5NDY0ZjU1MGZlZjg3MTQiLCJwIjoiaiJ9
https://binnenland.atlassian.net/browse/GN-1973?atlOrigin=eyJpIjoiYzg4NmUxYzYxMjM0NDVhY2ExNTQ4YWUxNmVmZDVhMmQiLCJwIjoiaiJ9

And cleans a little bit the code removing all the functions are files that are not used in the new flow

File: support/notulen-queries.js [hunk 1]
```diff
@@ -24,13 +24,13 @@ async function getZittingForNotulen(uuid) {
...
         besluit:geplandeStart ?geplandeStart;
-        prov:startedAtTime ?startedAt;
-        prov:endedAtTime ?endedAt;
         mu:uuid ${sparqlEscapeString(uuid)}.
       ?bestuursorgaanUri mandaat:isTijdspecialisatieVan ?mainBestuursorgaanUri.
       ?mainBestuursorgaanUri skos:prefLabel ?bestuursorgaanName.
       OPTIONAL {
-        ?uri prov:atLocation ?location.
+        ?uri prov:atLocation ?location;
+          prov:startedAtTime ?startedAt;
```
Comment (nvdk) [inline support/notulen-queries.js:32]: probably best to wrap each of these in an optional statement as each of them can be missing

File: support/behandeling-exporter.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
+// @ts-ignore
 import { uuid, query, update, sparqlEscapeUri, sparqlEscapeString } from 'mu';
-import {handleVersionedResource, cleanupTriples, hackedSparqlEscapeString} from './pre-importer';
-import {findFirstNodeOfType, findAllNodesOfType} from '@lblod/marawa/dist/dom-helpers';
-import { analyse, resolvePrefixes } from '@lblod/marawa/dist/rdfa-context-scanner';
+import {handleVersionedResource, hackedSparqlEscapeString} from './pre-importer';;
```
Comment (nvdk) [inline support/behandeling-exporter.js:3]: double semi

**Review by nvdk — CHANGES_REQUESTED** [review]
some small things to fix

**Pull request merged**